packages/backend/src/projectAdapters/dbtGitProjectAdapter.ts

```diff
@@ -44,8 +45,18 @@ const gitErrorHandler = (e: Error) => {
             { message: e.message },
         );
     }
+    if (e.message.includes('Repository not found')) {
+        const repo = e.message.match(/github\.com\/([^/]+\/[^/.]+)/)?.[1]; // Get repo from error message
```

Comment:
This will fail when they override the domain or when using Gitlab, bit bucket. 
<img width="409" alt="Screenshot 2024-11-12 at 16 29 41" src="https://github.com/user-attachments/assets/3ed7cf07-2711-4928-8722-218715ff6b12">

Reply:
remoteRepositoryUrl is the full url , including token 

> Could not find Git repository "https://lightdash:xxxx@github.com/rephus/jaffle-shop.git". Check that your personal access token has access to the repository and that the repository name is correct.

Built from 

>         const remoteRepositoryUrl = `https://${username}:${personalAccessToken}@${
            hostDomain || DEFAULT_HOST_DOMAIN
        }/${repository}.git`;

what we want is just the repository in the config, but this is not accessible, let me try improving the regex instead...

Reply:
done